Json/Json.sol

```diff
@@ -0,0 +1,31 @@
+pragma ton-solidity >= 0.47.0;
+import "../libraries/JsonLib.sol";
```

Comment:
we need load it from github like:
import "https://raw.githubusercontent.com/tonlabs/DeBot-IS-consortium/main/libraries/JsonLib.sol";
it will be more clear

Reply:
OK